code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -469,12 +469,12 @@
 	name = "Overload Machine"
 	desc = "Overheats a machine, causing a small explosion after a short time."
```

Comment:
With the new explosion size, would you still consider it small or its better to change the description?

Reply:
Changed to "Moderately-sized".